src/Http/ApiProblem/ApiProblem.php

```diff
@@ -543,7 +543,7 @@ public static function imageMustBeLinkedToResource(string $imageId): self
         );
     }
 
-    public static function invalidJsonData(string $detail): self
+    public static function invalidJsonDataForRdfCreation(string $detail): self
     {
         return self::create(
             'https://api.publiq.be/probs/body/invalid-json-data',
```

Comment:
Should this URL match the new naming better?

Reply:
Change the URI.
However this error is not documented if you actually follow the URL, would you like me to document this error or is this not needed because internal?

Reply:
This is indeed an internal error, so I would not expose it for now.